.github/dependabot.yml

```diff
@@ -0,0 +1,28 @@
+version: 2
+updates:
+  - package-ecosystem: "github-actions"
+    directory: "/"
+    schedule:
+      interval: "weekly" # default = monday
+    open-pull-requests-limit: 30
+    target-branch: "main"
+    labels:
+      - "dependencies"
+      - "github"
+  - package-ecosystem: "gradle"
+    directory: "/"
+    schedule:
+      interval: "daily"
```

Comment:
Is it intended that the interval for github-actions is _weekly_, whereas for gradle and docker _daily_?

Reply:
yes, as GitHub Actions will have no/low security related impact to the application, weekly is sufficient